### PR TITLE
Interactive keynote announcements.

### PR DESCRIPTION
**Pull request by mikeal**

### CODE REVIEW

File: locale/en/blog/announcements/interactive-2015-keynotes.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Node.js Interactive brings together a broad range of speakers to help experienced and novice Node.js users alike learn tips, best practices, new skills, as well as gain insight into future developments for the technology.  
+
+2015 Node.js Interactive keynotes include:
+
+### Day 1, December 8, 2015
+
+* Jason Gartner, Vice President, WebSphere Foundation and PureApplication Dev at IBM
+* James Snell, Open Technologies at IBM, “Enterprise Adoptions Rates and How They Benefit the Community”
+* Joe McCann, Co-Founder and CEO at NodeSource, “Convergence: Evolving Node.js with Open Governance and an Open Community”
```
Comment (jasnell) [inline locale/en/blog/announcements/interactive-2015-keynotes.md:24]: These are incorrect. I'm giving the Convergence keynote :-)

Comment (mikeal) [inline locale/en/blog/announcements/interactive-2015-keynotes.md:24]: Whoops, I'll swap it and let them know the press release is off.

File: locale/en/blog/announcements/interactive-2015-keynotes.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* Ashley Williams, Developer Community and Content Manager at npm
+* Tom Croucher, Engineer Manager at Uber, “Node.js at Uber”
+
+### Day 2, December 9, 2015
+
+* Mikeal Rogers, Node.js Foundation Community Manager at The Linux Foundation, “Node.js Foundation Growth and Goals”
+* Danese Cooper, Distinguished Member of Technical Staff - Open Source at PayPal and Node.js Foundation Chairperson
+* Panel Discussion with Node.js Foundation [Technical Steering Committee members](https://nodejs.org/en/foundation/tsc/)
```
Comment (Fishrock123) [inline locale/en/blog/announcements/interactive-2015-keynotes.md:32]: TSC or CTC?

Comment (mikeal) [inline locale/en/blog/announcements/interactive-2015-keynotes.md:32]: It's a TSC panel, although the membership is still technical exactly the same :)

Comment (jasnell) [inline locale/en/blog/announcements/interactive-2015-keynotes.md:32]: btw, do we know how many TSC members are planning to be there?

**Comment by jasnell** [comment]
LGTM once the keynotes are fixed :-)

**Comment by mikeal** [comment]
@jasnell it got pushed already and then another push fixed them :)

File: locale/en/blog/announcements/interactive-2015-keynotes.md (hunk 2 above)
Comment (mikeal) [inline locale/en/blog/announcements/interactive-2015-keynotes.md:32]: we don't have the most accurate count on that yet, but enough for a panel for sure :)

**Pull request merged**